prov/verbs/src/verbs_info.c

```diff
@@ -1306,6 +1321,7 @@ static int fi_ibv_get_matching_info(uint32_t version,
 	struct fi_info *fi, *tail;
 	int ret, i;
 	uint8_t got_passive_info = 0;
+	enum fi_log_level level = FI_LOG_INFO;
```

Comment:
I think you can set this as level = prefer_xrx ? WARN : INFO, which would avoid checking every loop iteration below